<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>622</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>622</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>622</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetControlPanel" native="true">
    <property name="geometry">
     <rect>
      <x>791</x>
      <y>-10</y>
      <width>221</width>
      <height>544</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="layoutControlPanelMain">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="buttonOpen">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="buttonJpg">
           <property name="text">
            <string>JPG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBmp">
           <property name="text">
            <string>BMP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonGif">
           <property name="text">
            <string>GIF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxOrtho">
         <property name="text">
          <string>Parallel projection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>167</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>167</height>
          </size>
         </property>
         <property name="title">
          <string>Lines</string>
         </property>
         <widget class="QRadioButton" name="radioButtonStripple">
          <property name="geometry">
           <rect>
            <x>82</x>
            <y>28</y>
            <width>79</width>
            <height>23</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Stripple</string>
          </property>
         </widget>
         <widget class="QFrame" name="frameLineColor">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>62</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1000</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonLineColor">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>57</y>
            <width>60</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButtonSolid">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>28</y>
            <width>65</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Solid</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBoxLineWidth">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>60</width>
            <height>25</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonPointColor">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>123</y>
            <width>60</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
         <widget class="QFrame" name="framePointColor">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>128</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1000</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButtonPoint">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>94</y>
            <width>79</width>
            <height>23</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Point</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBoxPointWidth">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>126</y>
            <width>60</width>
            <height>25</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBoxRound">
          <property name="geometry">
           <rect>
            <x>83</x>
            <y>94</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Round</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>68</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>68</height>
          </size>
         </property>
         <property name="title">
          <string>Background</string>
         </property>
         <widget class="QPushButton" name="buttonBGColor">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>29</y>
            <width>60</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
         <widget class="QFrame" name="frameBFColor">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>34</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="radioButtonMove">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Move</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonRotate">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Rotate</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonZoom">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Zoom</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxStep">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayoutDirection" columnstretch="0,0,0">
           <item row="1" column="0">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="2">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="buttonUp">
             <property name="minimumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>↑</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="buttonRight">
             <property name="minimumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>→</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="buttonDown">
             <property name="minimumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>↓</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="buttonLeft">
             <property name="minimumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>←</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="buttonZ1">
             <property name="minimumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>↻</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="buttonZ2">
             <property name="minimumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>27</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>↺</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
